File: server_client_C_sharp/mpp_c_app1/FestivalModel/Angajat.cs

using System;

namespace FestivalModel
{

    [Serializable]

    public class Angajat : Persoana
    {
        public Angajat(long idPersoana, string nume, string prenume) : base(idPersoana, nume, prenume)
        {
            idPersoana = idPersoana;
            nume = nume;
            prenume = prenume;
        }

        public String Username { get; set; }
        public String Password;

        public Angajat() : base()
        {
        }

        public String Oficiu { get; set; }


        public String toString()
        {
            return "Angajat{" +
                   "username='" + Username + '\'' +
                   ", oficiu='" + Oficiu + '\'' +
                   //   ", nume='" + nume  + '\'' +
                   //  ", prenume='" + getPrenume() + '\'' +
                   '}';
        }
    }
}

File: server_client_java_protoV3/JavaSocketsFestival_proto/Java11ChatSockets/ChatModel/src/main/java/festival/model/Angajat.java
package festival.model;
import java.io.Serializable;


public class Angajat extends Persoana implements Identifier<Long>, Serializable {

    private String username;

    public Angajat() {

    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    private String password;
    private String oficiu;


    public Angajat(Long id_persoana, String nume, String prenume) {
        super(id_persoana, nume, prenume);
    }
    public Angajat(String usern, String parola) {
        username=usern;
        password=parola;
    }

    @Override
    public Long getId() {
        return super.getId();
    }

    @Override
    public void setId(Long id_angajat) {
        super.setId(id_angajat);
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getOficiu() {
        return oficiu;
    }

    public void setOficiu(String oficiu) {
        this.oficiu = oficiu;
    }

    @Override
    public String toString() {
        return "Angajat{" +
                "username='" + username + '\'' +
                ", oficiu='" + oficiu + '\'' +
                ", nume='" + getNume() + '\'' +
                ", prenume='" + getPrenume() + '\'' +
                '}';
    }
}